### PR TITLE
Update milestone test to reflect peril-settings' trunk

### PR DESCRIPTION
**Pull request by mokagio**
See https://github.com/Automattic/peril-settings/pull/49/files

There's no direct way to test this yet, because I'm yet to setup a Danger + GitHub Actions workflow that uses the milestone `Dangerfile`.

The tests should cover us, and I'll obviously test all the permutations when I'll get to set up the workflow, so I think this is ok for now.

### CODE REVIEW

File: org/pr/milestone.ts [hunk 1]
```diff
@@ -15,19 +15,34 @@ declare var danger: DangerDSLType
 export declare function warn(message: string): void
 
 export async function checkMilestone() {
```
Comment (mokagio) [inline org/pr/milestone.ts:17]: This is a pure copy paste of the work from https://github.com/Automattic/peril-settings/pull/49/files

👇 👇 👇

File: tests/pr/milestone.test.ts [hunk 2]
```diff
@@ -0,0 +1,117 @@
+import { checkMilestone as milestone } from "../org/pr/milestone";
```
Comment (mokagio) [inline tests/pr/milestone.test.ts]: Unfortunately, the changes to this file were too many and with the move it just looks like a completely new file in the diff.

That might not be a bad things, actually. Gives a better chance to review it.

Unlike the production code, this one is a bit different, due to the way we need to write `Dangerfile`s by importing from the `node_modules/` at runtime and therefor mock in the `global` context in the tests.

**Review by oguzkocer — APPROVED** [review]
This looks pretty much what we have in `peril-settings` repo, as you also noted, so :shipit: 

I am going to leave the merge to you @mokagio since you said there are no testing steps we can follow and you'll run some tests, I want to make sure you can merge it when you're ready. :)

**Comment by mokagio** [comment]
Thanks @oguzkocer 😄 

I'm going to merge this now, unfortunately there isn't a way to run Danger with remote `Dangerfile` with imports that are not on `master`, so we need it merged in order to test it.

**Pull request merged**